x-pack/plugins/ml/public/maps/anomaly_layer_wizard.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { LAYER_WIZARD_CATEGORY } from '../../../maps/common';
+import type { LayerWizard } from '../../../maps/public';
+
+export const anomalyLayerWizard: Partial<LayerWizard> = {
+  categories: [LAYER_WIZARD_CATEGORY.SOLUTIONS],
```

Comment:
I'd also add the Elasticsearch-category, so it shows up in that facet as well

```
categories: [LAYER_WIZARD_CATEGORY.SOLUTIONS, LAYER_WIZARD_CATEGORY.ELASTICSEARCH],
```
![image](https://user-images.githubusercontent.com/1833023/150019482-24c0f581-125d-42de-8950-cd64997cea14.png)

Reply:
Added in e121d6724193fd765c79cc6fb03af95cb4377510